src/relay/pass/pass_manager.cc

```diff
@@ -64,14 +64,18 @@ Pass GetPass(const std::string& pass_name) {
     return FoldScaleAxis();
   } else if (pass_name == "PartialEvaluate") {
     return SimplifyInference();
+  } else if (pass_name == "QuantizeAnnotate") {
```

Comment:
Could we open an issue to follow up with this and add a pass registry, similar to the op registry?

Reply:
Will do together with the O level system.